privacyidea/lib/token.py

```diff
@@ -140,7 +140,7 @@ def create_tokenclass_object(db_token):
 def _create_token_query(tokentype=None, realm=None, assigned=None, user=None,
                         serial_exact=None, serial_wildcard=None, active=None, resolver=None,
                         rollout_state=None, description=None, revoked=None,
-                        locked=None, userid=None, tokeninfo=None, maxfail=None):
+                        locked=None, userid=None, tokeninfo=None, maxfail=None, filterRealm=None):
```

Comment:
Also here, I would prefer to rename the parameter to something more descriptive.

Reply:
yes, yes, we will get rid of this "filterRealm". Everywhere!

Reply:
done in 80747ff6